portal-web/test/functional/com/liferay/portalweb/paths/portlet/ckeditorsample/CKEditorBalloonEditorSample.path

```diff
@@ -0,0 +1,63 @@
+<html>
+<head>
+<title>CKEditorBalloonEditorSample</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+	<tr><td rowspan="1" colspan="3">CKEditorBalloonEditorSample</td></tr>
+</thead>
+
+<tbody>
+
+<!--PARAGRAPH-->
+<tr>
+	<td>BODY_TEXT</td>
+	<td>//div[@class='portlet-body']/div/div/p[${paragraph_number}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BODY_TEXT_HYPERLINK</td>
+	<td>//div[@class='portlet-body']/div/div/p[2]/a | //div[@class='portlet-body']/div/div/p/a</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_ASSERT_BODY_TEXT_CONTENT_BOLD</td>
+	<td>//div[(@class='portlet-body')]//p//strong[contains(text(),'${key_textName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_ASSERT_BODY_TEXT_CONTENT_BULLET_LIST</td>
+	<td>//div[(@class='portlet-body')]//ul/li[contains(text(),'${key_textName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_ASSERT_BODY_TEXT_CONTENT_ITALIC</td>
+	<td>//div[(@class='portlet-body')]//p//em[contains(text(),'${key_textName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_ASSERT_BODY_TEXT_CONTENT_UNDERLINE</td>
+	<td>//div[(@class='portlet-body')]//p//u[contains(text(),'${key_textName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PARAGRAPH_CONTENT_NUMBERED_LIST</td>
+	<td>//div[(@class='portlet-body')]//ol/li</td>
+	<td></td>
+</tr>
+<tr>
+	<td>TABLE</td>
+	<td>//div[contains(@class,'editable')]//table/tbody</td>
```

Comment:
Does `cke_editable` element work? I think that's the pattern I see for CKEditor paths.